app/views/matches/index.html.erb

```diff
@@ -1,26 +1,72 @@
-<p id="notice"><%= notice %></p>
+<%= render "shared/header" %>
 
-<h1>Matches</h1>
-
-<table>
-  <thead>
-    <tr>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @matches.each do |match| %>
-      <tr>
-        <td> <%= match.away_team.name %> @ <%= match.home_team.name %> </td>
-        <td><%= link_to 'Show', match %></td>
-        <td><%= link_to 'Edit', edit_match_path(match) %></td>
-        <td><%= link_to 'Destroy', match, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New Match', new_match_path %>
+<div class="container">  
+  <div class="spacer"></div>
+  <div class="matches_container open_sans_font" style="padding-top: 50px;">
+    <div class="top_container">
+      <div class="title_container">
+        <h1>UPCOMING MATCHES</h1>
```

Comment:
No need to fix this, but in future projects it would be ideal to add a CSS class that adds upper-casing to the element contents instead of hardcoding uppercasing all text.